test/test_autograd.py

```diff
@@ -3921,6 +3921,17 @@ def test_inputbuffer_add_multidevice(self, devices):
         output = input.to(device=devices[1]) + input.to(device=devices[1])
         output.backward()
 
+    def test_copy_(self, device):
+        # At the time of writing this test, copy_ is not generated from native_functions.yaml
+        # there was a bug that bfloat16 was not recognized as floating.
+        x = torch.randn(10, device=device, requires_grad=True)
+        for dt in [torch.float, torch.double, torch.half, torch.bfloat16]:
```

Comment:
maybe better to loop though https://github.com/pytorch/pytorch/blob/3d745508eb85112932ac6baba45408ecd58c18bc/torch/testing/__init__.py#L89 and just invoke the test on everything that has is_floating_point set?

Reply:
Oh, thanks! I had only grepped in test/ and not found it...